src/main/java/filter_post/use_case/FilterPostInteractor.java

```diff
@@ -31,7 +31,7 @@ public void filterPosts(FilterPostRequestModel filters) {
 
         for (String[] postInfo : posts) {
             String postTags = postInfo[4];
-            if (containsAllWords(postTags, filterTags)) {
+            if (filterTags.length == 0 || containsAllWords(postTags, filterTags)) {
```

Comment:
Redundant condition. `containsAllWords` already returns true if `filterTags` is empty.

Reply:
Got it, will update